pkgs/development/python-modules/hdbscan/default.nix

```diff
@@ -0,0 +1,32 @@
+
+{ lib
+, buildPythonPackage
+, cython
+, numpy
+, scipy
+, scikitlearn
+, pythonOlder
+, fetchPypi
+}:
+
+buildPythonPackage rec {
+  pname = "hdbscan";
+  version = "0.8.12";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0yxi34frg2jwyvjl942qy4gq5pbx8dq4pf4p28d1xah8njchfqir";
+  };
+
+  # No tests in archive
+  doCheck = false;
```

Comment:
Can we fetch from GitHub instead and use something like `nostetests -s hdbscan` as the `checkPhase`?